crates/pink/src/runtime/extension.rs

```diff
@@ -323,4 +357,22 @@ impl PinkExtBackend for CallInCommand {
     fn is_running_in_command(&self) -> Result<bool, Self::Error> {
         Ok(true)
     }
+
+    fn ecdsa_sign_prehashed(
```

Comment:
Why do we offer prehash signing only for ecdsa?

Reply:
The other two don't call a hash before passing the payload into their inner sign().

Reply:
Could we double check? Usually they should accept 256 bits pre-hashed message, but apparently they accept a `Vec<u8>` instead. So I guess some hash is still used.

Reply:
Double checked.